### PR TITLE
enhance and modernize gemspec

### PR DESCRIPTION
**Pull request by noraj**

### CODE REVIEW

File: twitter_cldr.gemspec [hunk 1]
```diff
@@ -1,28 +1,40 @@
-$:.unshift File.join(File.dirname(__FILE__), 'lib')
-require 'twitter_cldr/version'
+# frozen_string_literal: true
+
+require_relative 'lib/unisec/version'
```
Comment (camertron) [inline twitter_cldr.gemspec:3]:
```suggestion
require_relative 'lib/twitter_cldr/version'
```

**Review by camertron — CHANGES_REQUESTED** [review]
Nice, thanks!

File: twitter_cldr.gemspec (hunk 1 above)
Comment (noraj) [reply twitter_cldr.gemspec:3]: I was maybe a bit too quick on the copy paste 😅

**Comment by noraj** [comment]
About the CLA : I have the same email address attached to two github accounts so the app want me to sign it with both accounts.

**Review by camertron — APPROVED** [review]

**Pull request merged**